libcflib/db.xsh

```diff
@@ -49,17 +51,15 @@ class DB:
         self._channel_graphs = {}
         self._packages = {}
         self._initialized = True
+        self._idx = $LIBCFGRAPH_INDEX
 
-    def _build_whoosh(self):
-        self.idx = 'whoosh'
-
-    def search(self, **kwargs):
+    def search(self, query):
         """Search the database
 
         Parameters
         ----------
-        kwargs : dict
-            The keys to search on
+        query: dict
+            The keys to search on. e.g. `{"about.conda_version": "4.2.13"}`
```

Comment:
Does this need double '`'?

Reply:
Does it? I thought that single backticks were handled correctly, though I'm not sure.

Reply:
Im not certain either, maybe someone else can help with this. I thought RST was two but idk if this is true RST.